cvat/apps/engine/static/engine/js/annotationParser.js

```diff
@@ -77,7 +79,8 @@ class AnnotationParser {
 
         let occluded = +shape.getAttribute('occluded');
         let z_order = shape.getAttribute('z_order') || '0';
-        return [points, occluded, +z_order];
+        let client_id = box.getAttribute('client_id') || '-1'
```

Comment:
@azhavoro ,

Need to check somehow that client_id is valid. For example, we don't have the same client id for different boxes or all of them are -1. The check should be on client side when we parse annotation and on server side when the user presses down key.